configuration/config-txt/video.md

```diff
@@ -621,6 +621,10 @@ Use `display_lcd_rotate` to rotate or flip the LCD orientation. Parameters are t
 
 Use `display_rotate` to rotate or flip the screen orientation. Parameters are the same as `display_hdmi_rotate`.
 
+### disable_fw_kms_setup 
+
+By default, the firmware parses the EDID of any HDMI attached display, picks an appropriate video mode, then passes the resolution and frame rate of the mode, along with overscan parameters, to the Linux kernel via settings on the kernel command line. You can use `disable_fw_kms_setup=1` to disable the passing of these parameters. The Linux video mode system (KMS) will then parse the EDID itself and pick an appropriate mode.
```

Comment:
> via settings on the kernel command line.

This can have the effect of creating a mode that is not present in the EDID, and which may be incompatible with the monitor.

> You can use `disable_fw_kms_setup=1` to disable the passing of these parameters, and thereby avoid this problem.